docs/reference/supervisor.md

```diff
@@ -1991,7 +1991,7 @@ namespace webots {
 from controller import Supervisor
 
 class Supervisor (Robot):
-    def setLabel(self, id, label, xpos, ypos, size, color, transparency, font="Arial"):
+    def setLabel(self, id, label, x, y, size, color, transparency=0, font='Arial'):
```

Comment:
Why the transparency has a default value in the Python API but not in the C++ API?

Reply:
I agree, this is inconsistent. I will add this default argument to the C++ and Java APIs...